web/styles/right_sidebar.css

```diff
@@ -484,13 +484,33 @@ $user_status_emoji_width: 24px;
     }
 }
 
-.right-sidebar-shortcuts {
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
+.invite-user-shortcut {
     /* The margin top is calculated from a legacy 25px height,
        from a 20px line of text and 5px of margin top. We calculate
        a scaling margin-top by subtracting the em-unit line height
        from the legacy value. */
     margin-top: calc(25px - (var(--legacy-body-line-height-unitless) * 1em));
+    margin-right: var(--width-simplebar-scroll-hover);
+    padding-left: var(--right-sidebar-left-margin);
```

Comment:
This makes me think `margin` maybe shouldn't be in the variable name for this chunk of space?

Reply:
Yeah fair, I'll rename it to `spacing`.